sentry_sdk/worker.py

```diff
@@ -8,7 +8,7 @@
 
 MYPY = False
 if MYPY:
-    from queue import Queue
+    from queue import Queue  # type: ignore # for --py2
```

Comment:
If we do this we might as well import the queue from `_compat`, which already imports the right queue depending on version

Reply:
Yes, `queue` is actually imported from `_compat` just above, but I think mypy gets confused and prefers the stdlib. I tried various things, couldn't get it to work, so just added the ignore. It's only needed for py2 though.